pkg/config/config.go

```diff
@@ -277,18 +277,18 @@ func (idCfg *IdentityConfig) validateAndInit() (err error) {
 	// to the kube and kubelet APIs. So, we might end up getting an X.509 certificate with the old pod IP.
 	// To avoid this, we fail the current run with an error to force SYNC the status on the pod resource and let
 	// the subsequent retry for the init container to attempt to get a new certificate from the identity provider.
-	if idCfg.Init && err == nil && idCfg.ProviderService != "" {
-		log.Errorf("SIA(init) detected the existence of X.509 certificate at %s", idCfg.CertFile)
+	if idCfg.Init && err == nil && idCfg.ServiceCert.CopperArgos != nil {
```

Comment:
```suggestion
	if idCfg.Init && err == nil && idCfg.providerService != ""{
```
`validateAndInit`の方が、`loadDerivedConfig`より先に呼ばれると思います

Reply:
ty